test/uk/gov/hmrc/taxhistory/services/helper/EmploymentHistoryServiceHelperSpec.scala

```diff
@@ -113,121 +115,103 @@ class EmploymentHistoryServiceHelperSpec extends PlaySpec with MockitoSugar with
       merged.allowances.size mustBe 0
       merged.allowances mustBe Nil
 
-      merged.benefits.get.size mustBe 1
-      val benefits1 = merged.benefits.get(employment1.employmentId.toString)
+      merged.benefits.size mustBe 1
+      val Some(benefits1) = merged.benefits.get(employment1.employmentId.toString)
       benefits1.size mustBe 1
       benefits1 must contain(companyBenefit)
 
-      merged.payAndTax.get.size mustBe 1
-      val payAndTax1 = merged.payAndTax.get(employment1.employmentId.toString)
+      merged.payAndTax.size mustBe 1
+      val Some(payAndTax1) = merged.payAndTax.get(employment1.employmentId.toString)
```

Comment:
I've not seen this syntax when declaring a val before, can you explain it?

Specifically, it's this part: val Some(payAndTax1) = ...

What happens when the value assigned to it is None?

Reply:
It's very much like pattern matching. When the match fails (in this case, when it's None), an exception is thrown (same as before the changes, when there was an `Option.get`)
It doesn't make it safe, just arguably clearer (and it was the quicker way to change the test so it compiled again). But this is a test, in production we shouldn't do it, we should check for None.